### PR TITLE
fix: fix wordy message

### PR DESCRIPTION
**Pull request by britaniar**
### Description of your changes

Fixes #

I have:
Quote the cluster names.

- [x] Run `make reviewable` to ensure this PR is ready for review.

### Special notes for your reviewer

Fix bug bash bug:
Change 'britaniar-aks-member-3 is not selected: specified cluster is not found'
to "Specified cluster "xxx" is not found"

### CODE REVIEW

File: pkg/scheduler/framework/framework.go [hunk 1]
```diff
@@ -41,7 +41,7 @@ const (
 	// The reasons to use for scheduling decisions.
 	pickedByPolicyReason                  = "picked by scheduling policy"
 	pickFixedInvalidClusterReasonTemplate = "cluster is not eligible for resource placement yet: %s"
-	pickFixedNotFoundClusterReason        = "specified cluster is not found"
+	pickFixedNotFoundClusterReason        = "specified cluster \"%s\" is not found"
```
Comment (ryanzhang-oss) [inline pkg/scheduler/framework/framework.go:44]: reason is a machine read code, it should not be templated.

File: pkg/controllers/clusterresourceplacement/placement_status.go [hunk 2]
```diff
@@ -142,6 +142,9 @@ func (r *Reconciler) setResourceConditions(ctx context.Context, crp *fleetv1beta
 		if unselected[i].ClusterScore != nil {
 			scheduledCondition.Message = fmt.Sprintf(resourcePlacementConditionScheduleFailedWithScoreMessageFormat, unselected[i].ClusterName, unselected[i].ClusterScore, unselected[i].Reason)
 		}
+		if unselected[i].Reason == fmt.Sprintf("specified cluster \"%s\" is not found", unselected[i].ClusterName) {
```
Comment (ryanzhang-oss) [inline pkg/controllers/clusterresourceplacement/placement_status.go:145]: just copy the reason should be enough

File: pkg/scheduler/framework/framework.go (hunk 1 above)
Comment (zhiying-lin) [reply pkg/scheduler/framework/framework.go]: the reason here is not the one in the condition, more like a message. I think using template should be fine.

File: pkg/controllers/clusterresourceplacement/placement_status.go (hunk 2 above)
Comment (zhiying-lin) [reply pkg/controllers/clusterresourceplacement/placement_status.go]: I would suggest to copy reason directly from the cluster decision. It means we will build the whole message in the scheduler instead.

**Review by ryanzhang-oss — APPROVED** [review]

File: pkg/scheduler/framework/framework.go [hunk 3]
```diff
@@ -40,9 +40,9 @@ const (
 
 	// The reasons to use for scheduling decisions.
 	pickedByPolicyReason                  = "picked by scheduling policy"
-	pickFixedInvalidClusterReasonTemplate = "cluster is not eligible for resource placement yet: %s"
-	pickFixedNotFoundClusterReason        = "specified cluster is not found"
-	notPickedByScoreReason                = "cluster does not score high enough"
+	pickFixedInvalidClusterReasonTemplate = "cluster \"%s\" is not eligible for resource placement yet: %s"
```
Comment (zhiying-lin) [inline pkg/scheduler/framework/framework.go:43]: Message is usually using upper case. Can you fix that?

eg deployment condition,

status:
  conditions:
  - lastTransitionTime: "2024-05-21T08:31:21Z"
    lastUpdateTime: "2024-05-21T08:31:24Z"
    message: ReplicaSet "member-agent-76c4999765" has successfully progressed.
    reason: NewReplicaSetAvailable
    status: "True"
    type: Progressing
  - lastTransitionTime: "2024-05-24T14:53:37Z"
    lastUpdateTime: "2024-05-24T14:53:37Z"
    message: Deployment does not have minimum availability.
    reason: MinimumReplicasUnavailable
    status: "False"
    type: Available

File: pkg/scheduler/framework/framework.go [hunk 4]
```diff
@@ -40,9 +40,9 @@ const (
...
+	pickFixedNotFoundClusterReason        = "specified cluster \"%s\" is not found"
```
Comment (zhiying-lin) [inline pkg/scheduler/framework/framework.go:44]: should be "pickFixedNotFoundClusterReasonTemplate"

File: pkg/scheduler/framework/framework.go (hunk 3 above)
Comment (britaniar) [reply pkg/scheduler/framework/framework.go:43]: Addressed on #833

**Pull request merged**